### PR TITLE
Html emails

### PR DESCRIPTION
**Pull request by azogue**
**Description:**
- Add the ability of sending rich html emails, not only text emails with
attached images.
- Customize the `product_name` to be included as sender in emails.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#7313

### CODE REVIEW

**Comment by Landrash** [comment]
@azogue Which PR are the current parent PR?

File: source/_components/notify.smtp.markdown [hunk 1]
```diff
@@ -79,6 +81,73 @@ To use the SMTP notification, refer to it in an automation or script like in thi
 
 The optional `images` field adds in-line image attachments to the email. This sends a text/HTML multi-part message instead of the plain text default.
 
+The optional `html` field makes a custom text/HTML multi-part message, allowing total freedom for sending rich html emails. In them, if you need to attach images, you can pass both arguments (`html` and `images`), the attachments will be joined with the basename of the images, so they can be included in the html page with `src="cid:image_name.ext"`.
+
+```yaml
```
Comment (Landrash) [inline source/_components/notify.smtp.markdown]: Add ` # Example configuration.yaml entry``in the top of this example.

File: source/_components/notify.telegram.markdown [hunk 2]
```diff
@@ -18,7 +18,7 @@ The `telegram` platform uses [Telegram](https://web.telegram.org) to delivery no
 The requirements are:
 
 - You need a [Telegram bot](https://core.telegram.org/bots). Please follow those [instructions](https://core.telegram.org/bots#6-botfather) to create one and get the token for your bot. Keep in mind that bots are not allowed to contact users. You need to make the first contact with your user. Meaning that you need to send a message to the bot from your user.
-- The `chat_id` of an user.
+- The `chat_id` of an user, or a list of `user_id`s for sending messages to multiple users.
```
Comment (Landrash) [inline source/_components/notify.telegram.markdown]: Separate changes to telegram out to it's own PR.

**Comment by azogue** [comment]
I'm closing this PR and I will do it again without the telegram changes, sorry
